vendor/src/github.com/docker/swarmkit/manager/manager.go

```diff
@@ -184,7 +195,7 @@ func New(config *Config) (*Manager, error) {
 			}
 			if proto == "unix" && unwrappedErr == syscall.EADDRINUSE {
 				os.Remove(addr)
```

Comment:
Probably not an issue, since `win.ListenPipe` calls `CreateFile`, which should never return `EADDRINUSE`. If it did, this `os.Remove` call would try to remove the named pipe with `DeleteFile`, which will fail. The pipe will not be deleted until the last handle is closed. Therefore, this code path does not apply to Windows.